### DISCUSSION TITLE
Import snATAC data in HDF5 format

### DISCUSSION THREAD

**Discussion by Nobuyuki-Enzan** [GreenleafLab/ArchR#1855] [Questions / Documentation] [answered]
Hi ArchR team,

Is there any way to create arrow files from scATAC data (HDF5 format).
The multiome-tutorial (https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html) shows how to import h5 files,
but this is for scRNAseq data.

The script below successfully import the HDF5 file 
(default of "featureType" is "Gene Expression", but I have to set as "Peaks" since my HDF5 data is snATAC data).
inputdata <- import10xFeatureMatrix(input=path_vector, names=names_vector, featureType="Peaks")
However, there is no tutorial about how to create arrowfiles/ArchRproject from this "inputdata".

Best,
Nobuyuki

**Comment by rcorces** [comment, marked as answer]
No, this is not supported and never will be. The matrix-based HDF5 files from CellRanger do not contain the information required to create an Arrow file. Arrow files should ideally be created from fragment files but could also be created from BAM files.

**Comment by Nobuyuki-Enzan** [comment]
Thank you for your quick response.

Please let me ask just one more thing.
Same thing is true for other formats (e.g. seurat.rds format)?
I mean, we can not create Arrow files from dataset created by other software?

**Reply by rcorces** [reply]
the only acceptable input formats for ATAC-seq in ArchR are fragment files or BAM files

**Reply by Nobuyuki-Enzan** [reply]
Thank you for your assistance.
